components/SearchBar/index.js

```diff
@@ -0,0 +1,76 @@
+import { styled } from "styled-components";
```

Comment:
```suggestion
import styled from "styled-components";
```
The official documentation of `styled-components` says that `styled` is a default export.